### PR TITLE
COMMERCE-5268 added successful product image

### PR DESCRIPTION
**Pull request by garofalog**

### CODE REVIEW

**Comment by garofalog** [comment]
ci:test:sf

**Comment by garofalog** [comment]
ci:test:sf

**Comment by garofalog** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-product-definitions-web/src/main/resources/META-INF/resources/edit_attachment_file_entry.jspf [hunk 1]
```diff
@@ -129,23 +129,27 @@ if (cpAttachmentFileEntry != null) {
 		});
 	});
 
-	function <portlet:namespace />saveAttachmentFileEntry(forceDisable) {
-		var form = window.document['<portlet:namespace />fm'];
+	Liferay.provide(
+		window,
+		'<portlet:namespace />saveAttachmentFileEntry',
+		function (forceDisable) {
```
Comment (gianmarcobrunialti) [inline modules/apps/commerce/commerce-product-definitions-web/src/main/resources/META-INF/resources/edit_attachment_file_entry.jspf:135]: This `forceDisable` is an arg that doesn't seem like it's used anywhere. Can we remove it? @garofalog

**Review by gianmarcobrunialti — COMMENTED** [review]
Non-blocking comment, looks ok to me!

**Comment by garofalog** [comment]
ci:test:sf

**Comment by garofalog** [comment]
ci:test:sf

**Comment by garofalog** [comment]
ci:test:relevant

**Comment by gianmarcobrunialti** [comment]
@garofalog Tests are failing due to semantic versioning issues. I think a simple
```
blade gw baseline -Dbaseline.ignoreFailures=true
```
will do the job (run the command in your branch, and commit everything that it changes under with the appropriate commit message, e.g. `COMMERCE-5268 Semantic versioning (gw baseline)`).

/cc @andrea-ale-sbarra to make sure, the error message looks weird to me anyway.

**Comment by garofalog** [comment]
ci:reopen

**Comment by garofalog** [comment]
ci:test:relevant

**Comment by garofalog** [comment]
ci:reopen

**Comment by garofalog** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:forward

**Pull request closed without merging**